site/en/docs/extensions/mv3/promises/index.md

```diff
@@ -55,10 +61,10 @@ Not all methods in extensions APIs support promises. Sometimes that's because we
 promise support on the method yet; in many cases it's because using a promise isn't feasible for the
 method.
 
-You see if an API method support promises by checking its API reference page:
- 
-{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/AYQVtnh19vNMHoXzxZB1.png", alt="Screenshot showing a
-method that supports promises", width="800", height="280" %}
+You can check whether a method supports promises by checking its API reference page:
+
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/AYQVtnh19vNMHoXzxZB1.png",
+  alt="Screenshot showing a method that supports promises", width="800", height="280" %}
 
 The screenshot above shows a method from the [`chrome.tabs`][api-tabs-methods] API. You can see that
```

Comment:
Rather than stating "The screenshot above", you can say:

> The example method, `captureVisibleTab`, can be found in the [`chrome.tabs`][api-tabs-methods] API.
> This method supports promises because one of the method's signatures returns a promise. ...

Reply:
Applying the suggested copy.

Reply:
Awesome, thank you!